e2e/process-services-cloud/people-group-cloud-component.e2e.ts

```diff
@@ -0,0 +1,141 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import TestConfig = require('../test.config');
+
+import { LoginSSOPage } from '../pages/adf/loginSSOPage';
+import { SettingsPage } from '../pages/adf/settingsPage';
+import { NavigationBarPage } from '../pages/adf/navigationBarPage';
+import { PeopleGroupCloudComponentPage } from '../pages/adf/demo-shell/process-services/peopleGroupCloudComponentPage';
+import { PeopleCloudComponent } from '../pages/adf/process-cloud/peopleCloudComponent';
+import { GroupCloudComponent } from '../pages/adf/process-cloud/groupCloudComponent';
+import { browser } from 'protractor';
+import { Identity } from '../actions/APS-cloud/identity';
+import { GroupIdentity } from '../actions/APS-cloud/groupIdentity';
+import CONSTANTS = require('../util/constants');
+
+describe('People Groups CLoud Component', () => {
+
+    describe('People Groups CLoud Component', () => {
+        const settingsPage = new SettingsPage();
+        const loginSSOPage = new LoginSSOPage();
+        const navigationBarPage = new NavigationBarPage();
+        const peopleGroupCloudComponentPage = new PeopleGroupCloudComponentPage();
+        const peopleCloudComponent = new PeopleCloudComponent();
+        const groupCloudComponent = new GroupCloudComponent();
+        const identityService: Identity = new Identity();
+        const groupIdentityService: GroupIdentity = new GroupIdentity();
+
+        let silentLogin;
+        let apsUser;
+        let activitiUser;
+        let noRoleUser ;
+        let selectedPeople;
+        let groupAps;
+        let groupActiviti;
+        let groupNoRole;
+        let selectedGroups;
+
+        beforeAll(async () => {
+            await identityService.init(TestConfig.identity.adminUser, TestConfig.identity.adminPassword);
+            apsUser = await identityService.createIdentityUser();
+            await identityService.assignRole(apsUser.id, CONSTANTS.ROLES.APS_USER);
+            activitiUser = await identityService.createIdentityUser();
+            await identityService.assignRole(activitiUser.id, CONSTANTS.ROLES.ACTIVITI_USER);
+            noRoleUser = await identityService.createIdentityUser();
+            selectedPeople = [`${apsUser.firstName}` + ' ' + `${apsUser.lastName}`, `${activitiUser.firstName}` + ' ' + `${activitiUser.lastName}`,
+                `${noRoleUser.firstName}` + ' ' + `${noRoleUser.lastName}`];
+            await groupIdentityService.init(TestConfig.identity.adminUser, TestConfig.identity.adminPassword);
```

Comment:
Could you try to use a for loop for lines 62-67?

Reply:
I think, it will be too much of looping just to assign 2 roles.